autoendpoint/src/routers/fcm/client.rs

```diff
@@ -238,10 +238,17 @@ impl FcmClient {
         // Handle error
         let status = response.status();
         if status.is_client_error() || status.is_server_error() {
-            let data: FcmResponse = response
-                .json()
+            let raw_data = response
```

Comment:
Per the metrics, we want this change in `send_gcm`, not `send` (but it might as well happen in both places)

Reply:
ah, doh! thanks forgot about the `send_gcm` function.